doc/supported-hardware.md

```diff
@@ -54,6 +61,7 @@ The following table depicts the supported SR-IOV hardware features of each suppo
 | Mellanox MT28908 Family [ConnectX-6 Lx] | V | V | V |
 | Mellanox MT28908 Family [ConnectX-7] | V | V | V |
 | Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx | V | V | V |
+| Mellanox MT43244 BlueField-3 integrated ConnectX-6 Dx | V | V | V |
```

Comment:
Did we accidentally pull in another commit?

Reply:
Yeah I was a bit confused by this inclusion. Not sure if this was intentional. I will remove this for now, if we need to backport cx-7 support we can do this in a separate PR